lib/types/ServerMetadata.ts

```diff
@@ -0,0 +1,57 @@
+export interface ServerMetadata {
+  credential_endpoint: string;
+  credentials_supported: CredentialsSupported;
+  credential_issuer?: { display?: ClaimIssuerDisplay };
+}
+
+export interface CredentialsSupported {
+  [x: string]: {
+    display?: Display[];
+    formats: Formats;
+    claims: Claims;
+  };
+}
+
+export interface Display {
+  name: string;
+  locale?: string;
+  logo?: Logo;
+  [x: string]: unknown;
+}
+
+export interface Logo {
+  url?: string;
+  alt_text?: string;
+  description?: string;
+  background_color?: string;
+  text_color?: string;
+  [x: string]: unknown;
+}
```

Comment:
As per specs this should be in Display interface. And example is different ... I guess let's approve it.